modules/ROOT/pages/twilio/twilio-connector.adoc

```diff
@@ -8,9 +8,9 @@ _Select_
 
 Twilio is a cloud communications infrastructure as a service (IaaS).
 
-This connector provides an API for making and receiving telephone calls, and sending and receiving text messages. To get started with Twilio, follow the steps below to gain access to their free sandbox service to send SMS text-messages. You can configure the Twilio connector in Anypoint Studio with your API credentials.
+The Twilio platform serves APIs for text messaging, VoIP, and voice calls. The Anypoint Connector for Twilio provides connectivity to the Twilio text messaging API.
```

Comment:
If no voice, the first sentence in line 11 is incorrect: "The Twilio platform serves APIs for text messaging, VoIP, and voice calls."  -- Both VOIP and voice are DOA

Reply:
My understanding is that Twilio provides SMS and voice, and the Anypoint Connector for Twilio provides SMS but not voice. Previous versions incorrectly claimed that the Anypoint Connector for Twilio provides SMS and voice. Should I use another way to explain it in the new version?